<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchBox</class>
 <widget class="QWidget" name="SearchBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>57</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEdit_ReplaceWith"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_ReplaceWith">
     <property name="text">
      <string>Replace with:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_Find">
     <property name="text">
      <string>Find:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="SearchLineEdit" name="lineEdit_Find"/>
   </item>
   <item row="1" column="3">
    <widget class="QFrame" name="frame_FindButtons">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="toolButton_FindPrevious">
        <property name="text">
         <string>Find previous</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_FindNext">
        <property name="text">
         <string>Find next</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_SearchWhileTyping">
        <property name="text">
         <string>Search while typing</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QFrame" name="frame_ReplaceButtos">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="toolButton_Replace">
        <property name="text">
         <string>Replace</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_ReplaceAndFind">
        <property name="text">
         <string>Replace &amp;&amp; find</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_ReplaceAll">
        <property name="text">
         <string>Replace All</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="pushButton_hide">
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SearchLineEdit</class>
   <extends>QLineEdit</extends>
   <header>SearchLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
